crates/plugins/mediator-coordination/src/didcomm/bridge.rs

```diff
@@ -272,4 +285,122 @@ mod tests {
         let resolved = resolver.get_secret(secret_id).await.unwrap();
         assert!(resolved.is_none());
     }
+
+    #[tokio::test]
+    async fn test_local_did_resolver_resolves_peer_did_successfully() {
+
+        let diddoc = setup();
+        let resolver = LocalDIDResolver::new(&diddoc);
+
+        // Setup a sample Peer DID document
+        let peer_did_doc = r##"{
+                "id": "did:peer:0z6MkhaXgBZDvotDkL5257faiztiGiC2QtKLGpbnnEGta2doK",
+                "verificationMethod": [
+                    {
+                        "id": "#z6MkhaXgBZDvotDkL5257faiztiGiC2QtKLGpbnnEGta2doK",
+                        "type": "JsonWebKey2020",
+                        "controller": "did:peer:0z6MkhaXgBZDvotDkL5257faiztiGiC2QtKLGpbnnEGta2doK",
+                        "publicKeyJwk": {
+                            "kty": "OKP",
+                            "crv": "Ed25519",
+                            "x": "Lm_M42cB3HkUiODQsXRcweM6TByfzEHGO9ND274JcOY"
+                        }
+                    },
+                    {
+                        "id": "#z6LSj72tK8brWgZja8NLRwPigth2T9QRiG1uH9oKZuKjdh9p",
+                        "type": "JsonWebKey2020",
+                        "controller": "did:peer:0z6MkhaXgBZDvotDkL5257faiztiGiC2QtKLGpbnnEGta2doK",
+                        "publicKeyJwk": {
+                            "kty": "OKP",
+                            "crv": "X25519",
+                            "x": "bl_3kgKpz9jgsg350CNuHa_kQL3B60Gi-98WmdQW2h8"
+                        }
+                    }
+                ],
+                "authentication": ["#z6MkhaXgBZDvotDkL5257faiztiGiC2QtKLGpbnnEGta2doK"],
+                "keyAgreement": ["#z6LSj72tK8brWgZja8NLRwPigth2T9QRiG1uH9oKZuKjdh9p"],
+                "service": []
+            }"##;
+
+        // Resolving Peer DID
+        let did = "did:peer:0z6MkhaXgBZDvotDkL5257faiztiGiC2QtKLGpbnnEGta2doK";
+        let resolved = resolver.resolve(did).await.unwrap().unwrap();
+
+        // Expected DID Document
+        let expected: serde_json::Value = serde_json::from_str(peer_did_doc).unwrap();
+
+        // Debugging: Print both resolved and expected documents
+        println!(
+            "Resolved DID Doc: {}",
+            json_canon::to_string(&resolved).unwrap()
+        );
+        println!(
+            "Expected DID Doc: {}",
+            json_canon::to_string(&expected).unwrap()
+        );
```

Comment:
Why do you print the results?

Reply:
Any Suggestions ?

Reply:
Printing test results is not typically appropriate, as the primary goal is to verify that all tests pass. Printing can be useful for debugging during the development phase, but should be removed upon completion, leaving only the assertions in place.

Reply:
Ok

Reply:
@Hermann-Core Done ✅